repo/packages/B/beakerx/3/marathon.json.mustache

```diff
@@ -0,0 +1,119 @@
+{
+"id": "{{service.name}}",
+"cpus": {{service.cpus}},
+{{#service.gpu_support.enable}}
+"gpus": {{service.gpu_support.gpus}},
+{{/service.gpu_support.enable}}
+"mem": {{service.mem}},
+"instances": 1,
+"container": {
+"portMappings": [
+{
+"containerPort": 8888,
+"hostPort": 0,
+"labels": {
+"VIP_0": "/{{service.name}}:{{networking.port_api}}"
+},
+"protocol": "tcp",
+{{#networking.external_access.enable}}
+    "servicePort": {{networking.external_access.external_access_port}},
+{{/networking.external_access.enable}}
+"name": "beakerx"
+},
+{
+"containerPort": 8080,
+"hostPort": 0,
+"labels": {
+"VIP_0": "/{{service.name}}:8080"
+},
+"protocol": "tcp",
+"name": "nginx"
+}
+],
+"type": "MESOS",
+"volumes": [
+{{#storage.persistence.enable}}
+    {
+    "persistent": {
+    "type": "root",
+    "size": {{storage.host_volume_size}},
+    "constraints": []
+    },
+    "mode": "RW",
+    "containerPath": "persistent_data"
+    },
+    {
+    "containerPath": "/home/beakerx/persistent_data",
+    "hostPath": "persistent_data",
+    "mode": "RW"
+    }
+{{#service.gpu_support.mount_agent_libraries}}
+      ,{
+        "containerPath": "/opt/mesosphere",
```

Comment:
Do we mount the entire /opt/mesosphere somewhere else?

Reply:
We used to e.g. with Jenkins. But as discussed I remove this option in favor of providing the URI